api-docs/index.js

```diff
@@ -0,0 +1,101 @@
+/* global process */
+/* eslint import/no-nodejs-modules:0 */
+
+const fs = require('fs');
+const path = require('path');
+
+const JsonRefs = require('json-refs');
+const yaml = require('js-yaml');
```

Comment:
We'll want to add this to package.json as a devDependency too with `yarn add -D`